Tasks/PackerBuildV1/task.json

```diff
@@ -15,7 +15,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 13
+        "Patch": 14
```

Comment:
You will need to build the task after updating patch version. This will update the auto-generated file task.loc.json which you should commit.